### PR TITLE
Handle when the ICO file isn't present

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
# PR Summary

* Bump rcedit version used
* Check if ICO file exists before running rcedit

## PR Context

rcedit is failing to set icons in CI scenarios because the ICO doesn't exist in the package.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by iSazonov** [comment]
@TylerLeonhardt Does new version work? Can we merge?

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by TylerLeonhardt** [comment]
This needed more work because apparently the _actual_ issue is that the ICO doesn't exist. I'm tracking down why, but this change is nice because setting the ICO isn't too important.

cc @adityapatwardhan

**Comment by adityapatwardhan** [comment]
Setting of the icon should be done in packaging instead

**Comment by adityapatwardhan** [comment]
Changes should be done here: https://github.com/PowerShell/PowerShell/blob/d1aa11e367581736ffea570722cee93815c07a41/tools/packaging/packaging.psm1#L1618

**Comment by TravisEz13** [comment]
If we take this type of fix, I think we should do something more temporary like:  https://github.com/PowerShell/PowerShell/pull/13078/files

File: tools/install-powershell.ps1 [hunk 1]
```diff
@@ -423,15 +423,30 @@ try {
...
+                throw "$Destination\pwsh.exe not found. Please review the contents of the daily package."
+            }
+            (Test-Path "$Destination\assets\Powershell_avatar.ico") {
+                Write-Warning "ICO file ($Destination\assets\Powershell_avatar.ico) not found. Skipping running rcedit."
+                $skipRunning = $true
+                break
+            }
+            (Test-Path "~/.rcedit/rcedit-x64.exe") {
+                Write-Verbose "Install RCEdit for modifying exe resources" -Verbose
+                $rceditUrl = "https://github.com/electron/rcedit/releases/download/v1.1.1/rcedit-x64.exe"
```
Comment (TravisEz13) [inline tools/install-powershell.ps1:440]: Does this need to run on the client machine?

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

**Comment by TylerLeonhardt** [comment]
Closing this in favor of #13123

**Pull request closed without merging**